modules/installation-configuration-parameters.adoc

```diff
@@ -807,12 +807,19 @@ Optional AWS configuration parameters are described in the following table:
 |The AWS AMI used to boot compute machines for the cluster. This is required for regions that require a custom {op-system} AMI.
 |Any published or custom {op-system} AMI that belongs to the set AWS region. See _{op-system} AMIs for AWS infrastructure_ for available AMI IDs.
 
+|compute:
+  platform:
+    aws:
+      iamProfile:
+|The name of the IAM instance profile that you use for the machine. If you want the installation program to create the IAM instance profile for you, do not select iamProfile. You can specify iamProfile or iamRole, but you cannot specify both.
+|String
+
 |compute:
   platform:
     aws:
       iamRole:
-|A pre-existing AWS IAM role applied to the compute machine pool instance profiles. You can use these fields to match naming schemes and include predefined permissions boundaries for your IAM roles. If undefined, the installation program creates a new IAM role.
-|The name of a valid AWS IAM role.
+|The name of the IAM instance profile that you use for the machine. If you want the installation program to create the IAM instance profile for you, do not select  iamRole. You can specify iamRole or iamProfile, but you cannot specify both.
```

Comment:
```suggestion
|The name of the IAM instance role that you use for the machine. If you want the installation program to create the IAM instance role for you, do not select  iamRole. You can specify iamRole or iamProfile, but you cannot specify both.
```

I don't know if we need to mention it, but the installer creates an instance profile when you specify an IAM role (existing behavior).